### PR TITLE
Filters 2k7, 2k9, 3k6 added/modified

### PR DESCRIPTION
**Pull request by DD4WH**
first attempt filter addition/modification:
Added 2k7 and 2k9 bandpass (150Hz to fpass) & added lowpass option to the 3k6 filter.
73 de Frank DD4WH

### CODE REVIEW

File: mchf-eclipse/.settings/language.settings.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 			<provider copy-of="extension" id="org.eclipse.cdt.ui.UserLanguageSettingsProvider"/>
 			<provider-reference id="org.eclipse.cdt.core.ReferencedProjectsLanguageSettingsProvider" ref="shared-provider"/>
 			<provider-reference id="org.eclipse.cdt.managedbuilder.core.MBSLanguageSettingsProvider" ref="shared-provider"/>
-			<provider class="org.eclipse.cdt.managedbuilder.language.settings.providers.GCCBuiltinSpecsDetector" console="false" env-hash="1885161144660037189" id="ilg.gnuarmeclipse.managedbuild.cross.GCCBuiltinSpecsDetector" keep-relative-paths="false" name="CDT GCC Built-in Compiler Settings Cross ARM" parameter="${COMMAND} ${FLAGS} ${cross_toolchain_flags} -E -P -v -dD &quot;${INPUTS}&quot;" prefer-non-shared="true">
```
Comment (jkramarz) [inline mchf-eclipse/.settings/language.settings.xml]: May I suggest using http://stackoverflow.com/questions/1753070/git-ignore-files-only-locally ? :-)

Comment (DD4WH) [inline mchf-eclipse/.settings/language.settings.xml]: thanks for that! I changed my setup and hope it has been the last time to spoil the commits ;-)!
Hmmm, I just wanted to add the commits "new filters 2k7, 2k9, 3k6" and "fixed non-working switchoff of filters". I made a rebase before this afternoon, so everything should be as the latest version of Andreas. After that I added those two commits regarding filters.
Andreas, let me know whether that worked or if I need to alter my setup again.

**Comment by jkramarz** [comment]
@DD4WH, if you need help with squashing / rewording these commits to keep repo history clean, feel free to send me IM via Google Hangouts or Facebook Messenger. I'll be happy to help :-)

**Comment by DD4WH** [comment]
@Jakub: will try squashing, now that I know the word, I can google how to do it! Thanks for the PM offer!

@Andreas: commit still has a bug, filters work o.k., main menu works o.k., but calibration menu is broken. However, I did not touch it!?

**Comment by jkramarz** [comment]
@DD4WH, to hint you a bit more: `git rebase --interactive 3b92f9ebff79c61ae4d917118be895de83b95fb9` may be the exact command you're looking for :-)

**Comment by DD4WH** [comment]
Hi Andreas, hope you are well again!
did you fix the calibration menu, that was apparently broken after my filter mods? If yes, maybe you can find sometime to give me a hint what I was missing (maybe the menu has to have a number of menu entries dividable by 6 as an integer?).
Sorry that I was too early to submit this pull request! Hope the code is still o.k.
73 de Frank

**Comment by df8oe** [comment]
I have fixed it and reworked some other menu code, too. There are now only some small "oprical" issues which residee since some time there and are not related to your pull-request.

Good job - it's published as new daily snapshot!

vy 73
Andreas

**Pull request merged**